batch_finder_resource_method.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: api_reference
+title: Batch Finder Resource Method
+permalink: /batch_finder_resource_method
+excerpt: This documentation describes how restli framework support batch of search requirement.
+---
+
+# Batch Finder Resource Method
+
+## Contents
+
+-   [Overview](#overview)
+-   [Protocol](#protocol)
+    -   [Request](#request)
+    -   [Response](#response)
+-   [Client](#client)
+    -   [Java Request Builders](#java-request-builders)    
+-   [Resource API](#resource-api)
+    -   [Method Annotation and Parameters](#method-annotation-and-parameters)
+    -   [Criteria Filter](#criteria-filter)
```

Comment:
What do you think about this order?
 - CriteriaFilter
 - Method annotation
 - BatchFinder results
 - Error Handling
It will follow the steps the developer has to do to build a batch finder.